app/controllers/api/v2/config_groups_controller.rb

```diff
@@ -23,6 +23,8 @@ def show
       def_param_group :config_group do
         param :config_group, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
+          param :puppetclass_ids, Array, :desc => N_("IDs of associated Puppet classes")
+          param :puppetclass_names, Array, :desc => N_("Names of associated Puppet classes")
```

Comment:
I think we encountered this elsewhere recently too, we normally don't provide name keys in API and let hammer do it. While I agree this would be handy in API it would probably be better to do it for all resources at once. @tstrachota correct?